.travis-build.sh

```diff
@@ -26,7 +26,7 @@ if [[ "${GROUP}" != "cfi-tests" && "${GROUP}" != downstream* && "${GROUP}" != "a
   exit 1
 fi
 
-. ./.travis-build-without-test.sh
+. ./gradlew cloneAndBuildDependencies
```

Comment:
Doesn't that gradle target simply invoke the script?
Why is this indirection beneficial?
If this is necessary, please document it.

Reply:
@baoruiz Can you please resolve this issue so that we can merge this PR?